Mathlib/Algebra/Polynomial/Derivation.lean

```diff
@@ -148,4 +151,87 @@ theorem comp_aeval_eq (d : Derivation R A M) (f : R[X]) :
     _ = (AEval.of R M a).symm (d.compAEval a f) := rfl
     _ = _ := by simp [-compAEval_apply, compAEval_eq]
 
+end compAEval
+
+section coeffwise
+
+variable {R A M : Type*} [CommRing R] [CommRing A] [Algebra R A] [AddCommGroup M]
+  [Module A M] [Module R M] [IsScalarTower R A M] (d : Derivation R A M) (a : A)
+
+/--
+The `R`-derivation from `A[X]` to `M[X]` which applies the derivative to each
+of the coefficients.
+-/
+def coeffwise : Derivation R A[X] (PolynomialModule A M) where
```

Comment:
Could we have a more standard make?

Either `mapCoeffs` or `Derivation.polynomialModule`?

Reply:
Renamed to `mapCoeffs`.